build.gradle

```diff
@@ -51,13 +49,13 @@ buildscript {
  * 4. Comment 'wrapper' task
  */
 //task wrapper(type: Wrapper) {
```

Comment:
I think that the comment is no longer valid (at least not for gradle 2 and above) and the wrapper does not need to be commented out. It if is, the task still exists, but the version is derived from the used gradle version (I think..).